<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QMainWindow" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>551</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 163, 163)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit_ip">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_ip">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;URW Gothic L&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>IP Address</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chat">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Chat!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;URW Gothic L&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_port">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;URW Gothic L&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_title">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 32pt &quot;VL Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>C H A T</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_error">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListWidget" name="listWidget_users">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>421</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sendmsg">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>95</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255); text-color: &quot;black&quot;;</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_messages">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>31</y>
        <width>371</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
